source/features/pr-branches.tsx

```diff
@@ -67,13 +67,13 @@ function normalizeBranchInfo(data: BranchInfo): {
 	};
 }
 
-function buildQuery(numbers: number[]) {
+function buildQuery(issueIds: string[]) {
```

Comment:
No need to rename the variables, while not of type `number` they are still issue “numbers”

Reply:
But IMHO, `issueIds` make more sense and are relevant to the context. They are "issue" ids.

Reply:
It's actually `issue_1234` not just `1234` so I changed it to `issueIds`.

Reply:
🤦‍♂️ indeed, I thought they were just `1234`